client/src/pages/employer-table/index.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import EmployerTableComponent from "../../components/employer-table-component";
+
+function EmployerTable() {
```

Comment:
i was wondering if you can rename from "employer-table" to "employer-dashboard" and same goes with the "employer-table-component" to "employer-dashboard-component", just to keep things consistent